tests/unit_tests/pyanaconda_tests/core/test_profile.py

```diff
@@ -69,6 +69,36 @@
     ),
 ]
 
+WORKSTATIONPLUS_PARTITIONING = [
```

Comment:
You can do something like `WORKSTATION_PARTITIONING + [ ... ]` to make sure that `/` and `/home` are same.